modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -41,6 +42,110 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.13.0
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.0 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Update Symfony from 4.4.* to 5.4.*
+
+Symfony framework has been updated to LTS major version 5.4. This also affected a number of apps which required a new release and have been now included in the complete bundle.
+
+=== Drop setup of User Key encryption
+
+User Key encryption has been already deprecated in core versions later than 10.7. For this reason, the command line interface and web UI to enable User Key encryption is no longer available. If you are operating an ownCloud installation with User Key encryption enabled, please get in contact with support@owncloud.com to plan a migration to Master Key encryption. https://github.com/owncloud/encryption/pull/389[encryption#389]
+
+=== User Key encryption is no longer auto-enabled
+
+Executing the "encryption:encrypt-all" occ command will no longer auto-enable User Key encryption. https://github.com/owncloud/core/pull/40702[#40702]
+
+=== Added occ command to remove obsolete storages
+
+Metadata coming from storages are stored in the DB. Previously, when a storage has been removed from ownCloud that metadata remained in the DB. The new occ command allows to remove that metadata stored, reducing the amount of space used by the DB as well as slightly improving the performance since there will be less entries. https://github.com/owncloud/core/pull/40779[#40779]
+
+=== Enforce 2-factor authentication
+
+2-factor authentication can be enforced now. The feature requires at least an app implementing the 2-factor, otherwise no enforcement will be done. If the 2-factor authentication is enforced, all users will be required to use a 2-factor authentication app. Some specific groups selected by the admin can be excluded to let those users bypass the 2-factor authentication. https://github.com/owncloud/core/pull/40830[#40830]
```

Comment:
```suggestion
2-factor authentication can be enforced now. The feature requires at least one app implementing the 2-factor, otherwise no enforcement will be done. If the 2-factor authentication is enforced, all users will be required to use a 2-factor authentication app. Some specific groups selected by the admin can be excluded to let those users bypass the 2-factor authentication. https://github.com/owncloud/core/pull/40830[#40830]
```